docs/data/pages.ts

```diff
@@ -23,144 +23,171 @@ const pages: MuiPage[] = [
+          {
+            pathname: '/x/react-data-grid/columns',
+            children: [
+              { pathname: '/x/react-data-grid/column-definition' },
+              { pathname: '/x/react-data-grid/column-dimensions' },
+              { pathname: '/x/react-data-grid/column-visibility' },
+              { pathname: '/x/react-data-grid/custom-columns' },
+              { pathname: '/x/react-data-grid/column-header' },
+              { pathname: '/x/react-data-grid/column-menu' },
+              { pathname: '/x/react-data-grid/column-spanning' },
+              { pathname: '/x/react-data-grid/column-groups' },
+              { pathname: '/x/react-data-grid/column-ordering', plan: 'pro' },
+              { pathname: '/x/react-data-grid/column-pinning', plan: 'pro' },
+            ],
+          },
+          {
+            pathname: '/x/react-data-grid/rows',
+            children: [
+              { pathname: '/x/react-data-grid/row-definition' },
+              { pathname: '/x/react-data-grid/row-updates' },
+              { pathname: '/x/react-data-grid/row-height' },
+              { pathname: '/x/react-data-grid/row-spanning', newFeature: true },
+              { pathname: '/x/react-data-grid/master-detail', plan: 'pro' },
+              { pathname: '/x/react-data-grid/row-ordering', plan: 'pro' },
+              { pathname: '/x/react-data-grid/row-pinning', plan: 'pro' },
+              { pathname: '/x/react-data-grid/row-recipes', title: 'Recipes' },
+            ],
+          },
+          { pathname: '/x/react-data-grid/editing' },
+          { pathname: '/x/react-data-grid/sorting' },
+          {
+            pathname: '/x/react-data-grid/filtering-group',
+            title: 'Filtering',
+            children: [
+              { pathname: '/x/react-data-grid/filtering', title: 'Overview' },
+              { pathname: '/x/react-data-grid/filtering/customization' },
+              { pathname: '/x/react-data-grid/filtering/quick-filter' },
+              { pathname: '/x/react-data-grid/filtering/server-side', title: 'Server-side filter' },
+              { pathname: '/x/react-data-grid/filtering/multi-filters', plan: 'pro' },
+              {
+                pathname: '/x/react-data-grid/filtering/header-filters',
+                plan: 'pro',
+                newFeature: true,
+              },
+              { pathname: '/x/react-data-grid/filtering-recipes', title: 'Recipes' },
+            ],
+          },
+          { pathname: '/x/react-data-grid/pagination' },
+          {
+            pathname: '/x/react-data-grid/selection',
+            children: [
+              { pathname: '/x/react-data-grid/row-selection' },
+              { pathname: '/x/react-data-grid/cell-selection', plan: 'premium', newFeature: true },
+            ],
+          },
+          { pathname: '/x/react-data-grid/virtualization' },
+          { pathname: '/x/react-data-grid/accessibility' },
+          { pathname: '/x/react-data-grid/performance' },
```

Comment:
This could sit under **Resources**, it is more of a guide. There are also performance improving suggestions within the FAQs page. Perhaps they could also be merged?

Reply:
I'll move this to Resources here, and then follow up with a new PR to reevaluate the Performance and FAQ content.